spec/index.bs

```diff
@@ -7232,7 +7232,7 @@ dictionary GPURenderPassColorAttachment {
     Given a {{GPURenderPassColorAttachment}} |this|:
 
     1. Let |renderViewDescriptor| be |this|.{{GPURenderPassColorAttachment/view}}.{{GPUTextureView/[[descriptor]]}}.
-    1. Let |resolveViewDescriptor| be |this|.{{GPURenderPassColorAttachment/resolveTarget}}.{{GPUTextureView/[[texture]]}}.{{GPUTexture/[[descriptor]]}}.
+    1. Let |resolveViewDescriptor| be |this|.{{GPURenderPassColorAttachment/resolveTarget}}.{{GPUTexture/[[descriptor]]}}.
```

Comment:
this seems wrong: `resolveTarget` is a view, but it accesses it as `GPUTexture`

Reply:
oops, nice catch, thanks!